File: Swift-C++-Testsuite/test/arithmetic-operators/Reminder/Swift/SimpleReminder.swift
@main
struct MyMain {

    static func main() {
        let _ = simpleReminder(x: 43)
    }

    // We need to store, return, or use the result of the reminder to keep
    // the compiler to remove it during optimization.
    static func simpleReminder(x: Int) -> Int {
        var a = x
        var b = a % 2
        return b
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/SimpleReminder.cpp

struct MyMain {

static int main(){
    int result = simpleReminder(43);
    std::cout << result;
    return result;
}

// We need to store, return, or use the result of the reminder to keep
// the compiler to remove it during optimization.
static int simpleReminder(int x){
    int a = x;
    int b = a % 2;
    return b;
}
};

int main(){
    return MyMain::main();
}